Bloom.md

```diff
@@ -0,0 +1,604 @@
+---
+RFC: 4
+Status: Proposed
```

Comment:
```suggestion
Status: Accepted
```

Reply:
Based on the last meeting, this was accepted we were just waiting for more comments.